waterlog/src/Components/MonthlyWastage.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Bar, defaults } from 'react-chartjs-2';
+import { Globals } from './../Globals';
+
+const MonthlyWastageComponent = (props) => {
+  if(props.props.dataPoints){
+  var dataY = props.props.dataPoints.map(a => a.y);
```

Comment:
Indent is inconsistent as mentioned before choose one tab or two and stick to it